### PR TITLE
Adding cri reset script for resetting the states

### PR DESCRIPTION
**Pull request by abhinandanpb**
Adding a reset script to supplement kubeadm reset. Since kubeadm doesnt yet support complete toolset this will be acting as a supplement to kubeadm reset to clear the sandboxes, containers and images.

Signed-off-by: Abhinandan Prativadi <abhi@docker.com>

### CODE REVIEW

File: hack/cri-reset.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+CRICTL_CLI="crictl --runtime-endpoint /var/run/cri-containerd.sock --image-endpoint --/var/run/cri-containerd.sock"
```
Comment (Random-Liu) [inline hack/cri-reset.sh]: Still only runtime-endpoint is needed. :)

**Comment by Random-Liu** [comment]
LGTM with one nit.
@abhinandanpb Have you tried this script?

**Comment by abhinandanpb** [comment]
yes tried it successfully in addition to kubeadm reset

File: hack/cri-reset.sh (hunk 1 above)
Comment (abhinandanpb) [reply hack/cri-reset.sh]: arghhh

**Comment by abhinandanpb** [comment]
@Random-Liu done. PTAL

**Comment by Random-Liu** [comment]
LGTM

**Pull request merged**